scrapy/spiders/sitemap.py

```diff
@@ -9,6 +9,10 @@
 
 logger = logging.getLogger(__name__)
 
+# differentiating requests for use in child classes
+class SitemapIndexRequest(Request): pass
+class UrlSetRequest(Request): pass
+class RobotsRequest(Request): pass
```

Comment:
Could you please use boring pep8 style (`pass` on a new line, two lines between classes)? 

If you leave it as-is then over the next year or two someone will inevitably submit a pull request to fix code style; that'd be just a time waste for nothing :)